ui/app/models/ns-capabilities.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Fragment from 'ember-data-model-fragments/fragment';
+import { array } from 'ember-data-model-fragments/attributes';
+
+export default class NamespaceCapabilities extends Fragment {
+  @array('string') DisabledTaskDrivers;
+  @array('string') EnabledTaskDrivers;
```

Comment:
These in particular would be cool to see spelled out on pages like namespaces index

Reply:
Okay if I leave out of this PR to cut it off/get it shipped?

Reply:
yeah 100%!